packages/node_modules/@webex/internal-plugin-devices/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@webex/internal-plugin-devices",
+  "version": "1.80.41",
```

Comment:
This will need to be updated to whatever the latest version is (I think).

Reply:
CI should auto-roll the version, will update this once the PR is ready to merge.